src/main/java/ch/njol/skript/expressions/ExprExperienceCooldown.java

```diff
@@ -0,0 +1,98 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.Timespan;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Experience Cooldown")
+@Description({
+	"The experience cooldown of a player.",
+	"Experience cooldown is how long until a player can pick up another orb of experience."
+})
+@Examples({
+	"send experience cooldown of player",
+	"set experience cooldown of player to 1 hour",
+	"if experience cooldown of player >= 10 minutes:",
+		"\tclear experience cooldown of player"
+})
+@Since("INSERT VERSION")
+public class ExprExperienceCooldown extends SimplePropertyExpression<Player, Timespan> {
+
+	static {
+		register(ExprExperienceCooldown.class, Timespan.class, "[the] (experience|[e]xp) cooldown", "players");
+	}
+
+	@Override
+	@SuppressWarnings("unchecked")
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		setExpr((Expression<Player>) exprs[0]);
+		return true;
+	}
+
+
+	@Override
+	public @Nullable Timespan convert(Player player) {
+		return new Timespan(Timespan.TimePeriod.TICK, player.getExpCooldown());
+	}
+
+	@Override
+	public Class<?> @Nullable [] acceptChange(ChangeMode mode) {
+		switch (mode) {
+			case REMOVE_ALL, DELETE, REMOVE -> {return null;}
+		}
+		return CollectionUtils.array(Timespan.class);
```

Comment:
This should return null by default, and `Timespan.class` for actually intended `ChangeMode`s. Basically just do this the other way around